### ISSUE TITLE
question for hanikesn

### ISSUE THREAD

**Issue by Moscarda** [hanikesn/woff2otf#9]
@hanikesn message_content

sorry this is not the right place to message someone, but as github has eliminated the contact user feature, I didn't know where else to ask:

i was hoping you or another user could give me a tip on how to use this python script as a basis for another script, or a simple app, or automated workflow—that could handle batch conversions.

As a mac user, I usually drag and drop the script into Terminal and then drag and drop the woff file and press return.  This fills in file name and directory automatically, so theoretically, couldn't the script be modified to processes all woffs in a specified directory?  My programming skills are limited to basic html, css, javascript, etc... otherwise I would probably know how to do it myself.

Cheers,

@Moscarda | design@moscarda.com

**Comment by hanikesn** [comment]
This can easily be done with a simple one line shell script:
```
for file in *.woff; do woff2otf.py $file; done
```

**Comment by Moscarda** [comment]
Thanks for the quick reply, but can you dumb it down for me...like step...by...step 
(I'm really new to this!)

Would i type the shell script, then drag the woff2otf.py to Terminal, then enter the directory?
